keps/sig-release/3031-signing-release-artifacts/kep.yaml

```diff
@@ -0,0 +1,53 @@
+title: Signing release artifacts
+kep-number: 3031
+authors:
+  - "@saschagrunert"
+owning-sig: sig-release
+participating-sigs:
+  - sig-security
+status: provisional
+creation-date: 2021-11-02
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+##### WARNING !!! ######
+# prr-approvers has been moved to its own location
+# You should create your own in keps/prod-readiness
+# Please make a copy of keps/prod-readiness/template/nnnn.yaml
+# to keps/prod-readiness/sig-xxxxx/00000.yaml (replace with kep number)
+#prr-approvers:
+
+# see-also:
+#   - "/keps/sig-aaa/1234-we-heard-you-like-keps"
+#   - "/keps/sig-bbb/2345-everyone-gets-a-kep"
+# replaces:
+#   - "/keps/sig-ccc/3456-replaced-kep"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.23"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.23"
+  # beta: "v1.20"
+  # stable: "v1.22"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+# feature-gates:
+#   - name: MyFeature
+#     components:
+#       - kube-apiserver
+#       - kube-controller-manager
+# disable-supported: true
```

Comment:
I'd comment this out unless we know we'll have a feature gate.

Reply:
Done :+1: